CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+## Fleet 4.20.0 (Sep 8, 2022)
+
+* Improved vulnerability detection for macOS hosts by improving detection of Zoom, Ruby, and Node.js vulnerabilities. Warning: For users that download and sync Fleet's vulnerability feeds manually, there are [required adjustments](https://github.com/fleetdm/fleet/issues/6628) or else vulnerability processing will stop working. Users with the default vulnerability processing settings can safely upgrade without adjustments.
+
+* Fleet Premium: Improved the vulnerability automations by adding vulnerability scores (EPSS probability, CVSS score, and known exploit) to the webhook payload. Read more about vulnerability automations [here on fleetdm.com/docs](https://fleetdm.com/docs/using-fleet/automations#vulnerability-automations).
+
+* Add ability to know how many hosts, and which hosts, have Munki issues. This information is presented on the **Home > macOS** page and **Host details** page. This information is also available in the [`GET /api/v1/fleet/macadmins`](https://fleetdm.com/docs/using-fleet/rest-api#get-aggregated-hosts-mobile-device-management-mdm-and-munki-information) and [`GET /api/v1/fleet/hosts/{id}/macadmins`](https://fleetdm.com/docs/using-fleet/rest-api#get-hosts-mobile-device-management-mdm-and-munki-information) and API routes.
+
+* Fleet Premium: Added ability to test features, like software inventory, on canary teams by adding a [`features` section](https://fleetdm.com/docs/using-fleet/configuration-files#features) to the `teams` YAML document.
+
+* Added functionality to rotate the `token` in the `/device/{token}` URL used by Fleet Desktop every hour. Warning: This change is not compatible with older versions of Fleet Desktop. For users that manage Fleet Desktop updates, disable auto-updates, or pinned Fleet Desktop to an older version, we recommend manually updating Fleet Desktop and Orbit to a version greater than v1.0.0. Also, users with a `distributed_interval` higher than 10 minutes may experience a degraded experience. Users with the default settings can safely upgrade.
```

Comment:
@zhumo do you have any concerns with the "rotate the token` entry in the CHANGELOG?

Reply:
@noahtalerman This should be removed. We're holding this one back until we figure out how to do it in way we're happy with.

Reply:
> This should be removed.

Done ✅